larq_compute_engine/tests/end2end_test.py

```diff
@@ -154,14 +155,10 @@ def preprocess(data):
 
 
 def assert_model_output(model_lce, inputs, outputs):
-    for input, output in zip(inputs, outputs):
-        actual_outputs = run_model(model_lce, list(input.flatten()))
-        assert len(actual_outputs) > 1
-        np.testing.assert_allclose(actual_outputs[0], actual_outputs[1], rtol=1e-5)
-        for actual_output in actual_outputs:
-            np.testing.assert_allclose(
-                actual_output, output.flatten(), rtol=0.001, atol=0.25
-            )
+    interpreter = Interpreter(model_lce)
+    actual_outputs = interpreter.predict(inputs)
+    for actual_output, output in zip(actual_outputs, outputs):
+        np.testing.assert_allclose(actual_output[0], output, rtol=0.001, atol=0.25)
```

Comment:
This trailing, batch dimension in the output can be quite confusing. @Tombana do you think we should change that?

Reply:
I changed the behaviour to mimic the format of `keras.Model.predic` in a15db69fd1e0dd9ddfa613799b4d1453ef13a157